<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogListeMedecin</class>
 <widget class="QDialog" name="DialogListeMedecin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_rechercher">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>20</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>520</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_tri">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Id</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Specialite</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grade</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ressources</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Specialite</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Supprimermed">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ajoutermed">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Modifiermed">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableview">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>591</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
